docs/migration/2.0.0-migration.md

```diff
@@ -0,0 +1,101 @@
+# Bitops Core
+
+## Environment Variables
+Bitops is no longer using the `ENVIRONMENT` value, it instead uses; `BITOPS_ENVIRONMENT`
+
+Easiest migration solution; 
+Update startup command to read as followed;
+```
+docker run \
+-e BITOPS_ENVIRONMENT=$ENVIRONMENT   \    <-- If the ENVIRONMENT variable is being used. This is a simple approach. 
```

Comment:
This initially confused me when comparing against `docs/development-local.md` line 22